Packs/MicrosoftGraphSecurity/ReleaseNotes/2_2_26.md

```diff
@@ -0,0 +1,6 @@
+
+#### Modeling Rules
+
+##### Microsoft Graph Security Modeling Rules
+
+- Updated the Microsoft Graph Security modeling rules to enhance extraction logic for IPv4 and IPv6.
```

Comment:
Updated the Microsoft Graph Security modeling rules to enhance the extraction logic for IPv4 and IPv6.